ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -29,7 +29,13 @@ private[platform] object CompletionFromTransaction {
   ): CompletionStreamResponse =
```

Comment:
Why does this method not simply take a `CompletionInfo` as an argument? Then we'd have all the current and future extensions automatically available (submission ID, submission rank, ...) so that we can set the fields in the `Completion` message.

Reply:
This is easier to do in a subsequent PR, after the DB schemas have been updated.